docker/build_artifacts/sagemaker/serve.py

```diff
@@ -60,6 +62,7 @@ def __init__(self):
         # Use this to specify memory that is needed to initialize CUDA/cuDNN and other GPU libraries
         self._tfs_gpu_margin = float(os.environ.get("SAGEMAKER_TFS_FRACTIONAL_GPU_MEM_MARGIN", 0.2))
         self._tfs_instance_count = int(os.environ.get("SAGEMAKER_TFS_INSTANCE_COUNT", 1))
+        self._tfs_wait_time = int(os.environ.get("SAGEMAKER_TFS_WAIT_TIME", 600))
```

Comment:
Please make the time unit part of the environment variable name and the field name.

Reply:
sure, will add it in next revision.